tools/builder/react.js

```diff
@@ -171,6 +171,10 @@ function buildStatic( done ) {
 	global.window = window;
 	global.document = document;
 	global.navigator = window.navigator;
+	global.React = require( 'react' );
+	global.ReactDOM = require( 'react-dom' );
+	global.lodash = require( 'lodash' );
+	global.moment = require( 'moment' );
```

Comment:
Uh is this how we externalize these dependencies for the React dashboard :question:

Reply:
I believe we're using React to build some static HTML in a way I don't fully understand. Because these dependencies are now external, we need to make sure they're globals like they would be in the browser.

If you stop `yarn build-client` before it cleans up these files, you'll see `static.deps.json`:

```json
["lodash","moment","react"]
```

Maybe @zinigor can look this bit over 🙇

Reply:
Yes, we are using React in the build phase to create several HTML files that are used as loading placeholders. This allows us to use the same components (with limitations) as we do for the rest of the admin page. The method that prepares DOM and requires the JS file is here: https://github.com/Automattic/jetpack/blob/master/tools/builder/react.js#L166

I'm not sure how we're using WordPress dependencies here, but yes, we need to have all of these for the HTML builder entry point to be able to use them.